dvc/scm/git/__init__.py

```diff
@@ -287,9 +302,32 @@ def install(self):
                 "[ ! -d .git/rebase-merge ]",
             ],
             "checkout",
+            hook_path_fn,
+        )
+        self._install_hook("pre-commit", [], "status", hook_path_fn)
+        self._install_hook("pre-push", [], "push", hook_path_fn)
+
+        if use_pre_commit_tool:
+            self._integrate_pre_commit_tool()
+
+    def _integrate_pre_commit_tool(self):
+        if not which("pre-commit"):
+            raise DvcException("pre-commit is not installed")
+
```

Comment:
I wonder if we really need to run `pre-commit` for the user, maybe config is enough + a hint `now run \`pre-commit install\` to install git hooks`?